logstash-core/lib/logstash/pipeline.rb

```diff
@@ -696,10 +701,21 @@ def stalling_threads_info
       .each {|t| t.delete("status") }
   end
 
+  def collect_dlq_stats
+    dlq_metric = @metric.namespace([:stats, :pipelines, pipeline_id.to_s.to_sym, :dlq])
+    enabled = settings.get_value("dead_letter_queue.enable")
+    dlq_metric.gauge(:enabled, enabled)
+    if enabled
+      dlq_metric.gauge(:path, @dlq_writer.get_path.to_absolute_path)
```

Comment:
I used the metrics from the persistent queue (https://www.elastic.co/guide/en/logstash/current/node-stats-api.html#pipeline-stats) as a guide/model for the dlq metrics, which includes path.
Also, the path includes the pipeline_id, so, if I understand correctly, can differ with multi-pipelines.